src/libxrpl/ledger/View.cpp

```diff
@@ -1282,6 +1282,8 @@ addEmptyHolding(
         return tefINTERNAL;  // LCOV_EXCL_LINE
     if (view.peek(keylet::mptoken(mptID, accountID)))
         return tecDUPLICATE;
+    if (accountID == mptIssue.getIssuer())
+        return tesSUCCESS;
```

Comment:
is `tesSuccess` really the correct code to return here? From my intuition, an issuer should not be allowed to add an empty holding, therefore an error code should be returned instead - and the caller of `addEmptyHolding` should be responsible for checking that the accountID is not the issuer.

Reply:
I think this is the reason: https://github.com/XRPLF/rippled/pull/5875/files#r2426305173

Reply:
`temDST_IS_SRC` is the result returned by a `TrustSet` transaction if the account is the issuer. I think the situation is analogous enough that it would be appropriate here, too.